_spinnaker/using_dinghy.md

```diff
@@ -525,4 +534,4 @@ Dinghy supports all of the usual Go template conditionals. In addition to that,
     }
 {% endraw %}```
 
-*Note: the structure of the webhook data passed to Dinghy's template engine depends on the git service that is sending the webhook. This example uses a Github web hook.*
\ No newline at end of file
+*Note: the structure of the webhook data passed to Dinghy's template engine depends on the git service that is sending the webhook. This example uses a Github web hook.*
```

Comment:
First sentence:
- Capitalize 't' on "the"
- Capitalize 'g' on "git"
- "is sending" to 'sends' if there is a definite start/stop and it is not a situation where data is continuously streamed.
- It's not a webhook being sent, right? It's the payload. If so, add 'data' or 'payload' after 'webhook'

Second sentence:
- Capitalize 'h' in 'GitHub'
- web hook should be 1 word